src/ButtonPanel.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import Button from './Button';
+
+const numbers = [1, 2, 3, 4, 5, 6, 7, 8, 9, 0];
+const symbols = ['+', '-', '*', '/', '='];
+
+const ButtonPanel = () => (
+  <div>
+    <div>Button Panel</div>
+    <Button numbers={numbers} symbols={symbols} />
+  </div>
+);
```

Comment:
- [ ] it includes all the following buttons divided in the groups below: ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/buttonPanel/buttons.png))

* Group 1: AC, +/-, %, ÷
* Group 2: 7, 8, 9, X
* Group 3: 4, 5, 6, -
* Group 4: 1, 2, 3, +
* Group 5: 0, ., =


the `Button` component should be used **once** per each symbol